Xamarin.Forms.Platform.UAP/FormsCommandBar.cs

```diff
@@ -1,9 +1,7 @@
-﻿using System;
-using Windows.Foundation.Collections;
+﻿using Windows.Foundation.Collections;
```

Comment:
Small cleanup, but will be hard to make a  pr just for that, i just did it now since i was messing with the file anyway